_episodes/04-changes.md

```diff
@@ -613,6 +613,13 @@ Date:   Thu Aug 22 09:51:46 2013 -0400
 >    ~~~
 >    {: .language-bash}
 >
+>    Before moving on, we will commit these changes.
+>
+>    ---
```

Comment:
I think this (and line 620) needs to be changed from `---` to `~~~` for the git commit command to render properly.

Reply:
thanks.  it should be fixed now.

Reply:
That was so fast! Thank you!